src/v2i-hub/SpatPlugin/src/SpatPlugin.cpp

```diff
@@ -149,6 +142,61 @@ namespace SpatPlugin {
 			
 		}
 	}
+
+	void SpatPlugin::processSpat() {
+		PLOG(logDEBUG) << "Attempting to process packet as SPAT ...";
+		auto spatEncoded_ptr = std::make_shared<tmx::messages::SpatEncodedMessage>();
+		scConnection->receiveUPERSPAT(spatEncoded_ptr);
+		spatEncoded_ptr->set_flags(IvpMsgFlags_RouteDSRC);
+		spatEncoded_ptr->addDsrcMetadata(tmx::messages::api::msgPSID::signalPhaseAndTimingMessage_PSID);
+		auto rMsg = dynamic_cast<routeable_message *>(spatEncoded_ptr.get());
+		BroadcastMessage(*rMsg);
+	}
+
+	void SpatPlugin::processTSCBM() {
+		PLOG(logDEBUG) << "Starting BINARY SPaT Receiver ...";
+		auto spat_ptr = (SPAT*)calloc(1, sizeof(SPAT));
+		auto spatEncoded_ptr = std::make_shared<tmx::messages::SpatEncodedMessage>();
+		scConnection->receiveBinarySPAT(spat_ptr, PluginClientClockAware::getClock()->nowInMilliseconds());
```

Comment:
Do we need the DEBUG log if the `receiveBinarySPAT` function has similar DEBUG log?